From 707fb5982c66269f3335f5a8951f54bcf01df1f6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 20 Jan 2010 16:04:55 +0100
Subject: [PATCH] init: fixed emergency_shell argument parsing

---
 modules.d/99base/init | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/99base/init b/modules.d/99base/init
index 8dc00a9..ab033a7 100755
--- a/modules.d/99base/init
+++ b/modules.d/99base/init
@@ -35,7 +35,7 @@ wait_for_loginit()
 
 emergency_shell()
 {
-    if [ $1 = "-n" ]; then
+    if [ "$1" = "-n" ]; then
         _rdshell_name=$2
         shift 2 
     else
