From 05e42bd49b5b5b1b10ec9fc5830985105894963b Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 12 Feb 2010 12:19:33 +0100
Subject: [PATCH] dracut.conf: added add_dracutmodules

---
 dracut.conf | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dracut.conf b/dracut.conf
index 24dbbef..a39925a 100644
--- a/dracut.conf
+++ b/dracut.conf
@@ -6,6 +6,9 @@
 # Dracut modules to omit
 #omit_dracutmodules=""
 
+# Dracut modules to add to the default
+#add_dracutmodules=""
+
 # additional kernel modules to the default
 #add_drivers=""
 
