From aafa510eb08936958dc24f2fdb906218b3d89cd5 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 18 Feb 2010 17:28:41 +0100
Subject: [PATCH] dracut: removed "local" not inside of function

---
 dracut | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dracut b/dracut
index 4f9b5e3..47908c4 100755
--- a/dracut
+++ b/dracut
@@ -130,7 +130,6 @@ fi
 
 # source our config dir
 if [ "$confdir" ] && [  -d "$confdir" ]; then
-    local f
     for f in "$confdir"/*.conf; do 
 	[ -e "$f" ] && . "$f"
     done
