From e3e6814eee4239fb878f6136575b135f88ce51bc Mon Sep 17 00:00:00 2001
From: Philippe Seewer <philippe.seewer@bfh.ch>
Date: Fri, 26 Feb 2010 13:21:38 +0100
Subject: [PATCH] parse-kernel.sh must have a shebang

---
 modules.d/90kernel-modules/parse-kernel.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/90kernel-modules/parse-kernel.sh b/modules.d/90kernel-modules/parse-kernel.sh
index 59d7870..6278dac 100755
--- a/modules.d/90kernel-modules/parse-kernel.sh
+++ b/modules.d/90kernel-modules/parse-kernel.sh
@@ -1,3 +1,5 @@
+#!/bin/sh
+
 for p in $(getargs rdloaddriver=); do 
 	modprobe $p
 done
