From 0df77fb0e9e02ea71b24b1596d9897774393ef5e Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 19 Mar 2010 15:13:32 +0100
Subject: [PATCH] dracut.spec: remove libselinux, libsepol requirement

---
 dracut.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.spec b/dracut.spec
index 428dd16..ced8595 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -56,7 +56,7 @@ Requires: /bin/sh
 Requires: fileutils, gzip, tar
 Requires: lvm2 >= 2.02.33-9, dhclient
 Requires: filesystem >= 2.1.0, cpio, device-mapper, initscripts >= 8.63-1
-Requires: e2fsprogs >= 1.38-12, libselinux, libsepol, coreutils
+Requires: e2fsprogs >= 1.38-12, coreutils
 Requires: mdadm, elfutils-libelf 
 Requires(pre): plymouth >= 0.8.0-0.2009.29.09.19.1
 Requires: plymouth >= 0.8.0-0.2009.29.09.19.1
