From f180f9f344bba88d87ee3a210ff287b2c12da85e Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 16 Apr 2010 16:45:25 +0200
Subject: [PATCH] init: fix cdrom polling loop

---
 modules.d/99base/init | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/99base/init b/modules.d/99base/init
index 4696915..612c021 100755
--- a/modules.d/99base/init
+++ b/modules.d/99base/init
@@ -195,7 +195,7 @@ while :; do
 	# skip, if cdrom medium was already found
         strstr "$(udevadm info --query=env --path=${cdrom##/sys})" \
 	    ID_CDROM_MEDIA && continue
-        echo change > "$cdrom"        
+        echo change > "$cdrom/uevent"        
     done
     
     i=$(($i+1))
