From 7803caa833a3bfa7e1e1d7faba63a6a708c5c4aa Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 16 Apr 2010 17:03:34 +0200
Subject: [PATCH] dracut.spec: removed e2fsprogs requirement

---
 dracut.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.spec b/dracut.spec
index 22fb0fd..07232d1 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -56,7 +56,7 @@ Requires: /bin/sh
 Requires: fileutils, gzip, tar
 Requires: lvm2 >= 2.02.33-9, dhclient
 Requires: filesystem >= 2.1.0, cpio, device-mapper, initscripts >= 8.63-1
-Requires: e2fsprogs >= 1.38-12, coreutils
+Requires: coreutils
 Requires: mdadm, elfutils-libelf 
 Requires(pre): plymouth >= 0.8.0-0.2009.29.09.19.1
 Requires: plymouth >= 0.8.0-0.2009.29.09.19.1
