From d207cf3a7433d3c62959d29c54fe940a4bdccbe5 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 7 May 2010 10:16:03 +0200
Subject: [PATCH] dracut.spec: remove elfutils-libelf requirement

---
 dracut.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut.spec b/dracut.spec
index 07232d1..b55db9b 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -57,7 +57,7 @@ Requires: fileutils, gzip, tar
 Requires: lvm2 >= 2.02.33-9, dhclient
 Requires: filesystem >= 2.1.0, cpio, device-mapper, initscripts >= 8.63-1
 Requires: coreutils
-Requires: mdadm, elfutils-libelf 
+Requires: mdadm  
 Requires(pre): plymouth >= 0.8.0-0.2009.29.09.19.1
 Requires: plymouth >= 0.8.0-0.2009.29.09.19.1
 Requires: cryptsetup-luks
