From 619a609bea3b5adc7db8b0ba0fa1c62b407c1fec Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 26 May 2010 17:43:18 +0200
Subject: [PATCH] dracut-functions: beautified warnings

---
 dracut-functions | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dracut-functions b/dracut-functions
index c8ac6de..3cb1201 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -384,7 +384,7 @@ should_source_module() {
     "$1/check" $hostonly || return 1
     for dep in $("$1/check" -d); do
 	check_module_deps "$dep" && continue
-	dwarning "Cannot load $mod, dependencies failed."
+	dwarning "Cannot load dracut module \"$mod\", dependencies failed."
 	return 1
     done
 }
@@ -432,7 +432,7 @@ install_kmod_with_fw() {
 	    fi
 	done
 	if [[ $found != yes ]]; then
-	    dinfo "Possible missing firmware ${fw} for module ${mod}.ko"
+	    dinfo "Possible missing firmware \"${fw}\" for kernel module \"${mod}.ko\""
 	fi
     done
 }
