From a6688b76a63afa093f18b899a5864ec65a77102b Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 11 Jun 2010 12:52:02 +0200
Subject: [PATCH] crypt: install more aes modules

---
 modules.d/90crypt/installkernel | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/modules.d/90crypt/installkernel b/modules.d/90crypt/installkernel
index 7a58ea3..f61f501 100755
--- a/modules.d/90crypt/installkernel
+++ b/modules.d/90crypt/installkernel
@@ -1,2 +1,3 @@
 #!/bin/bash
-instmods dm_crypt cbc aes sha256 xts
+instmods dm_crypt cbc sha256 xts aes aes_generic aesni-intel aes-x86_64
+
