From 94d70b3724745fbcfa9e9f7f373da5252de820ff Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 22 Jun 2010 11:59:49 +0200
Subject: [PATCH] dracut-functions: use LC_ALL=C rather than LANG=C

---
 dracut-functions | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut-functions b/dracut-functions
index ee229f6..e627a50 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -218,7 +218,7 @@ inst_binary() {
     local LDSO NAME IO FILE ADDR I1 n f TLIBDIR
     [[ -e $initdir$target ]] && return 0
     # I love bash!
-    LANG=C ldd $bin 2>/dev/null | while read line; do
+    LC_ALL=C ldd $bin 2>/dev/null | while read line; do
 	[[ $line = 'not a dynamic executable' ]] && return 1
 	if [[ $line =~ not\ found ]]; then
 	    derror "Missing a shared library required by $bin."
