From 2df596227c805b46ac5808fef42d8661b46e0f79 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 12 Jul 2010 14:43:11 +0200
Subject: [PATCH] crypt/parse-crypt.sh: fix end label for luks udev rules

---
 modules.d/90crypt/parse-crypt.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90crypt/parse-crypt.sh b/modules.d/90crypt/parse-crypt.sh
index 1e8a659..775617f 100755
--- a/modules.d/90crypt/parse-crypt.sh
+++ b/modules.d/90crypt/parse-crypt.sh
@@ -17,7 +17,7 @@ else
 	echo 'ENV{ID_FS_TYPE}=="crypto_LUKS", RUN+="/sbin/initqueue --unique --onetime --name cryptroot-ask-%k /sbin/cryptroot-ask $env{DEVNAME} luks-$env{ID_FS_UUID}"' \
 	    >> /etc/udev/rules.d/70-luks.rules	
     fi
-    echo LABEL="luks_end" >> /etc/udev/rules.d/70-luks.rules
+    echo 'LABEL="luks_end"' >> /etc/udev/rules.d/70-luks.rules
 
 fi
 
