From d06ffa49ac700d8b88d4835e8c9d44d5397fccdc Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 23 Aug 2010 10:36:46 +0200
Subject: [PATCH] mkinitrd: do not call dracut in host only mode

---
 mkinitrd-dracut.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mkinitrd-dracut.sh b/mkinitrd-dracut.sh
index 64b3069..dd745d7 100644
--- a/mkinitrd-dracut.sh
+++ b/mkinitrd-dracut.sh
@@ -142,9 +142,9 @@ if [ -n "$img_vers" ]; then
 fi
 
 if [ -n "$basicmodules" ]; then
-	dracut -H $dracut_args --add-drivers "$basicmodules" "$target" "$kernel"
+	dracut $dracut_args --add-drivers "$basicmodules" "$target" "$kernel"
 else
-	dracut -H $dracut_args "$target" "$kernel"
+	dracut $dracut_args "$target" "$kernel"
 fi
 
 # vim:ts=8:sw=4:sts=4:et
