From 2f25c3f4a563063684f0cecdaa87790dfcf9c1f6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 8 Dec 2010 13:46:25 +0100
Subject: [PATCH] fcoe/parse-fcoe.sh: removed second else

---
 modules.d/95fcoe/parse-fcoe.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/modules.d/95fcoe/parse-fcoe.sh b/modules.d/95fcoe/parse-fcoe.sh
index 52def0a..f862131 100755
--- a/modules.d/95fcoe/parse-fcoe.sh
+++ b/modules.d/95fcoe/parse-fcoe.sh
@@ -73,7 +73,5 @@ else
         fi
         source /sbin/fcoe-genrules.sh
     done
-else
-    warn "Invalid arguments for fcoe=$fcoe"
 fi
 
