From cbea975b987fb994c0911c94ea6c83743dead60e Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 26 Jan 2011 13:53:43 +0100
Subject: [PATCH] fips: install .hmac files for cryptsetup and libs

---
 modules.d/01fips/install | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/modules.d/01fips/install b/modules.d/01fips/install
index 53f54de..31253cb 100755
--- a/modules.d/01fips/install
+++ b/modules.d/01fips/install
@@ -18,6 +18,13 @@ else
         /$libdir/libfreebl3.so /$libdir/libfreebl3.chk 
 fi
 
+if [ -e /sbin/.cryptsetup.hmac ]; then
+    for i in /sbin/.cryptsetup.hmac /$libdir/.libcryptsetup.so*.hmac; do
+        [ -e $i ] || continue
+        dracut_install $i
+    done
+fi
+
 dracut_install /usr/$libdir/hmaccalc/sha512hmac.hmac
 
 # vim:ts=8:sw=4:sts=4:et
