From f4dad880857d7c2f11d27cf8450071754e9a3ac7 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 28 Jan 2011 13:26:03 +0100
Subject: [PATCH] base: install "less" optionally

---
 modules.d/99base/install | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/modules.d/99base/install b/modules.d/99base/install
index c0dff8c..a317442 100755
--- a/modules.d/99base/install
+++ b/modules.d/99base/install
@@ -1,6 +1,7 @@
 #!/bin/bash
 dracut_install mount mknod mkdir modprobe pidof sleep chroot \
-    sed ls flock cp mv dmesg rm ln rmmod mkfifo less 
+    sed ls flock cp mv dmesg rm ln rmmod mkfifo
+dracut_install -o less
 if [ ! -e "${initdir}/bin/sh" ]; then
     dracut_install bash
     (ln -s bash "${initdir}/bin/sh" || :)
