From ca77f36f8e9995072de0309fb670765187e432bc Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 2 Feb 2011 14:48:37 +0100
Subject: [PATCH] fips: add aes-xts module

---
 modules.d/01fips/installkernel | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/01fips/installkernel b/modules.d/01fips/installkernel
index bce58e4..6cb2dc0 100755
--- a/modules.d/01fips/installkernel
+++ b/modules.d/01fips/installkernel
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-FIPSMODULES="aead aes_generic aes-x86_64 ansi_cprng cbc ccm chainiv ctr"
+FIPSMODULES="aead aes_generic aes-xts aes-x86_64 ansi_cprng cbc ccm chainiv ctr"
 FIPSMODULES="$FIPSMODULES des deflate ecb eseqiv hmac seqiv sha256 sha512"
 FIPSMODULES="$FIPSMODULES cryptomgr crypto_null tcrypt" 
 
