From 8c4549ff18aa27d46969db8deb32725be6ecaf35 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 2 Feb 2011 13:34:34 +0100
Subject: [PATCH] multipath: use new "-B" parameter

---
 modules.d/90multipath/multipathd.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules.d/90multipath/multipathd.sh b/modules.d/90multipath/multipathd.sh
index c9a2b3f..2f8460f 100755
--- a/modules.d/90multipath/multipathd.sh
+++ b/modules.d/90multipath/multipathd.sh
@@ -1,8 +1,8 @@
 #!/bin/sh
 
-if [ -e /etc/multipath.conf -a -e /etc/multipath/wwids ]; then
+if [ -e /etc/multipath.conf ]; then
     modprobe dm-multipath
-    multipathd
+    multipathd -B || multipathd
 else
     rm /etc/udev/rules.d/??-multipath.rules 2>/dev/null    
 fi
