From 8a6939656f5ce12157725eee6c8c21a83b3cbe56 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 8 Feb 2011 17:59:48 +0100
Subject: [PATCH] dracut-functions: write to $HOME/dracut.log instead of
 /tmp/dracut.log

---
 dracut-functions | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/dracut-functions b/dracut-functions
index 99e0c2e..597f8a7 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -28,8 +28,7 @@ strstr() { [[ $1 =~ $2 ]]; }
 if ! [[ $dracutlogfile ]]; then
     [[ $dracutbasedir = /usr/share/dracut ]] && \
 	dracutlogfile=/var/log/dracut.log || \
-	dracutlogfile=/tmp/dracut.log
-#    [[ -w $dracutlogfile ]] || dracutlogfile=/tmp/dracut.log
+	dracutlogfile=$HOME/dracut.log
     if [[ -w $dracutlogfile ]]; then
         >"$dracutlogfile"
     fi
