From c73bba03f8c7acd02e27590439c0aceae9fe496d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 25 Feb 2011 12:06:15 +0100
Subject: [PATCH] parse-biosdevname: put parenthesis around BIOSDEVNAME env var

---
 modules.d/97biosdevname/parse-biosdevname.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/97biosdevname/parse-biosdevname.sh b/modules.d/97biosdevname/parse-biosdevname.sh
index 09b0cec..6023fef 100755
--- a/modules.d/97biosdevname/parse-biosdevname.sh
+++ b/modules.d/97biosdevname/parse-biosdevname.sh
@@ -1,7 +1,7 @@
 # -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
 # ex: ts=8 sw=4 sts=4 et filetype=sh
 USE_BIOSDEVNAME=$(getarg biosdevname)
-if [ $USE_BIOSDEVNAME != "1" ]; then
+if [ "$USE_BIOSDEVNAME" != "1" ]; then
     udevproperty UDEV_BIOSDEVNAME=
     rm -f /etc/udev/rules.d/71-biosdevname.rules
 else
