From dc3066da86f6849b89dccd64f728d99e3db5b5a2 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 25 Feb 2011 12:47:50 +0100
Subject: [PATCH] dracut-functions: check if logfile dir is writable

---
 dracut-functions | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dracut-functions b/dracut-functions
index 597f8a7..681f095 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -29,8 +29,8 @@ if ! [[ $dracutlogfile ]]; then
     [[ $dracutbasedir = /usr/share/dracut ]] && \
 	dracutlogfile=/var/log/dracut.log || \
 	dracutlogfile=$HOME/dracut.log
-    if [[ -w $dracutlogfile ]]; then
-        >"$dracutlogfile"
+    if [[ -w ${dracutlogfile%/*} ]]; then
+        >>"$dracutlogfile"
     fi
 fi
 
