From 3539eec0c1dac902c517929dae6e9f4fcf87781c Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 25 Feb 2011 12:48:25 +0100
Subject: [PATCH] dracut-functions: add date to logfile messages

---
 dracut-functions | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/dracut-functions b/dracut-functions
index 681f095..d963ff8 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -36,17 +36,17 @@ fi
 
 dwarning() {
     echo "W: $@" >&2
-    [[ -w $dracutlogfile ]] && echo "W: $@" >>"$dracutlogfile"
+    [[ -w $dracutlogfile ]] && echo $(date) "Warn:" $@ >>"$dracutlogfile"
 }
 
 dinfo() {
     [[ $beverbose ]] && echo "I: $@" >&2
-    [[ -w $dracutlogfile ]] && echo "I: $@" >>"$dracutlogfile"
+    [[ -w $dracutlogfile ]] && echo $(date) "Info:" $@ >>"$dracutlogfile"
 }
 
 derror() {
     echo "E: $@" >&2
-    [[ -w $dracutlogfile ]] && echo "E: $@" >>"$dracutlogfile"
+    [[ -w $dracutlogfile ]] && echo $(date) "Err:" $@ >>"$dracutlogfile"
 }
 
 get_fs_env() {
