From 5a2034943da08007fa9f454d1cf4c5c448748c06 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 2 Mar 2011 14:59:01 +0100
Subject: [PATCH] fips/installkernel: turn off hostonly mode for fipsmodules

---
 modules.d/01fips/installkernel | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/01fips/installkernel b/modules.d/01fips/installkernel
index fc3c9bb..fb61b18 100755
--- a/modules.d/01fips/installkernel
+++ b/modules.d/01fips/installkernel
@@ -7,7 +7,7 @@ FIPSMODULES="$FIPSMODULES cryptomgr crypto_null tcrypt"
 mkdir -p "${initdir}/etc/modprobe.d"
 
 for mod in $FIPSMODULES; do 
-    if instmods $mod; then
+    if hostonly='' instmods $mod; then
         echo $mod >> "${initdir}/etc/fipsmodules"
         echo "blacklist $mod" >> "${initdir}/etc/modprobe.d/fips.conf"
     fi
