From e2f1c1f88ca4a1816cc7b6d7d8a666573b4ecb07 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 2 Mar 2011 14:59:36 +0100
Subject: [PATCH] fips/fips.sh: unbashify if clause

---
 modules.d/01fips/fips.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/01fips/fips.sh b/modules.d/01fips/fips.sh
index e0480cb..8efedc2 100755
--- a/modules.d/01fips/fips.sh
+++ b/modules.d/01fips/fips.sh
@@ -59,7 +59,7 @@ do_fips()
     return 0
 }
 
-if ! fipsmode=$(getarg fips) || [ $fipsmode == "0" ]; then
+if ! fipsmode=$(getarg fips) || [ $fipsmode = "0" ]; then
     rm -f /etc/modprobe.d/fips.conf >/dev/null 2>&1
 else
     set -e
