From b5b5ff12a81094673f118350fb3b219aeb7d1e43 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 2 Mar 2011 15:00:34 +0100
Subject: [PATCH] fips: install fipscheck

---
 modules.d/01fips/install | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/01fips/install b/modules.d/01fips/install
index 31253cb..4b3cd9f 100755
--- a/modules.d/01fips/install
+++ b/modules.d/01fips/install
@@ -26,5 +26,6 @@ if [ -e /sbin/.cryptsetup.hmac ]; then
 fi
 
 dracut_install /usr/$libdir/hmaccalc/sha512hmac.hmac
+dracut_install fipscheck
 
 # vim:ts=8:sw=4:sts=4:et
