From 44e90e5c5a7210e10cda1b975034ce99d3875e70 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 2 Mar 2011 15:01:03 +0100
Subject: [PATCH] install .hmac files if present

---
 dracut-functions         | 10 +++++++++-
 modules.d/01fips/install |  7 -------
 2 files changed, 9 insertions(+), 8 deletions(-)

diff --git a/dracut-functions b/dracut-functions
index 0cba333..5b85d84 100755
--- a/dracut-functions
+++ b/dracut-functions
@@ -168,13 +168,17 @@ inst_dir() {
 # Location of the image dir is assumed to be $initdir
 # We never overwrite the target if it exists.
 inst_simple() {
-    local src target
+    local src target 
     [[ -f $1 ]] || return 1
     src=$1 target="${2:-$1}"
     if ! [[ -d ${initdir}$target ]]; then
         [[ -e ${initdir}$target ]] && return 0
         inst_dir "${target%/*}"
     fi
+    # install checksum files also
+    if [[ -e  "${src%/*}/.${src##*/}.hmac" ]]; then 
+	inst "${src%/*}/.${src##*/}.hmac" "${target%/*}/.${target##*/}.hmac"
+    fi
     dinfo "Installing $src" 
     cp -pfL "$src" "${initdir}$target"
 }
@@ -186,6 +190,10 @@ inst_library() {
     local src=$1 dest=${2:-$1}
     [[ -e $initdir$dest ]] && return 0
     if [[ -L $src ]]; then
+	# install checksum files also
+        if [[ -e  "${src%/*}/.${src##*/}.hmac" ]]; then 
+            inst "${src%/*}/.${src##*/}.hmac" "${dest%/*}/.${dest##*/}.hmac"
+        fi
 	reallib=$(readlink -f "$src")
 	lib=${src##*/}
 	inst_simple "$reallib" "$reallib"
diff --git a/modules.d/01fips/install b/modules.d/01fips/install
index 4b3cd9f..9bc69c1 100755
--- a/modules.d/01fips/install
+++ b/modules.d/01fips/install
@@ -18,13 +18,6 @@ else
         /$libdir/libfreebl3.so /$libdir/libfreebl3.chk 
 fi
 
-if [ -e /sbin/.cryptsetup.hmac ]; then
-    for i in /sbin/.cryptsetup.hmac /$libdir/.libcryptsetup.so*.hmac; do
-        [ -e $i ] || continue
-        dracut_install $i
-    done
-fi
-
 dracut_install /usr/$libdir/hmaccalc/sha512hmac.hmac
 dracut_install fipscheck
 
