From 2af86a7080788798fd50f61f50c5d7f21c13eae1 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 2 Mar 2011 16:47:58 +0100
Subject: [PATCH] fips: hardcode install of libcrypto.* and libssl.*

---
 modules.d/01fips/install | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/modules.d/01fips/install b/modules.d/01fips/install
index 9bc69c1..8115706 100755
--- a/modules.d/01fips/install
+++ b/modules.d/01fips/install
@@ -18,6 +18,11 @@ else
         /$libdir/libfreebl3.so /$libdir/libfreebl3.chk 
 fi
 
+for i in /{usr,}/$libdir/libssl.so* /{usr,}/$libdir/libcrypto.so*; do
+    [ -e $i ] || continue
+    dracut_install $i
+done
+
 dracut_install /usr/$libdir/hmaccalc/sha512hmac.hmac
 dracut_install fipscheck
 
