From cfb4747e57fb59a65c995564f95b3b88e9209436 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 4 Mar 2011 11:32:58 +0100
Subject: [PATCH] lvm: move emergency script from 00 to 90

---
 modules.d/90lvm/parse-lvm.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90lvm/parse-lvm.sh b/modules.d/90lvm/parse-lvm.sh
index 5e93d3e..b0037e1 100644
--- a/modules.d/90lvm/parse-lvm.sh
+++ b/modules.d/90lvm/parse-lvm.sh
@@ -12,7 +12,7 @@ else
         {
             printf '[ -e "/dev/%s" ] || ' $dev
             printf 'warn "LVM "%s" not found"\n' $dev
-        } >> /emergency/00-lvm.sh
+        } >> /emergency/90-lvm.sh
     done
 fi
 
