From 0ce3429c6b5f86dfdf082fa3d720ffc1073449b4 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 4 Mar 2011 11:35:57 +0100
Subject: [PATCH] init: turn off "-e" mode in emergency

---
 modules.d/99base/init | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/99base/init b/modules.d/99base/init
index 0ea5134..81603b1 100755
--- a/modules.d/99base/init
+++ b/modules.d/99base/init
@@ -34,6 +34,7 @@ wait_for_loginit()
 
 emergency_shell()
 {
+    set +e
     if [ "$1" = "-n" ]; then
         _rdshell_name=$2
         shift 2 
