From 85d7ac93e94a2eeef1d4399585a4f9b399131f41 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 4 Mar 2011 11:36:35 +0100
Subject: [PATCH] init: die if /.die is present

---
 modules.d/99base/init | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/99base/init b/modules.d/99base/init
index 81603b1..6a4d280 100755
--- a/modules.d/99base/init
+++ b/modules.d/99base/init
@@ -46,6 +46,7 @@ emergency_shell()
     echo $@
     source_all emergency
     echo
+    [ -e /.die ] && exit 1
     if getarg rdshell || getarg rdbreak; then
         echo "Dropping to debug shell."
         echo
