From cda0d40758daecb7e79b0a82bcf94cfaedd38902 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 4 Mar 2011 11:37:19 +0100
Subject: [PATCH] init: display emergency warning with "warn()" instead of
 "echo"

---
 modules.d/99base/init | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/99base/init b/modules.d/99base/init
index 6a4d280..663316f 100755
--- a/modules.d/99base/init
+++ b/modules.d/99base/init
@@ -43,7 +43,7 @@ emergency_shell()
     fi
     wait_for_loginit
     echo ; echo
-    echo $@
+    warn $@
     source_all emergency
     echo
     [ -e /.die ] && exit 1
