From 6fa9f8b15065d1ccf3edea9febab88dae0f03dee Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 4 Mar 2011 11:38:08 +0100
Subject: [PATCH] init: cause a kernel panic, rather than sleep forever in
 emergency

---
 modules.d/99base/init | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/modules.d/99base/init b/modules.d/99base/init
index 663316f..a59d4c4 100755
--- a/modules.d/99base/init
+++ b/modules.d/99base/init
@@ -54,8 +54,9 @@ emergency_shell()
         [ -e /.profile ] || echo "exec 0<>/dev/console 1<>/dev/console 2<>/dev/console" > /.profile
         sh -i -l
     else
-        echo "Boot has failed, sleeping forever."
-        while :; do sleep 365d;done
+        warn "Boot has failed. To debug this issue add \"rdshell\" to the kernel command line."
+        # cause a kernel panic
+        exit 1
     fi
 }
 
@@ -207,7 +208,7 @@ while :; do
     
     i=$(($i+1))
     [ $i -gt $RDRETRY ] \
-        && { flock -s 9 ; emergency_shell "No root device found"; } 9>/.console_lock
+        && { flock -s 9 ; emergency_shell "No root device \"$root\" found"; } 9>/.console_lock
 done
 unset job
 unset queuetriggered
