From bea581515c940e10b89faa87bf4285a99164af31 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 9 Mar 2011 21:50:04 +0100
Subject: [PATCH] dracut: make "-i, --include SOURCE TARGET" work for files,
 too

---
 dracut | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/dracut b/dracut
index 80e5cc6..d271c56 100755
--- a/dracut
+++ b/dracut
@@ -264,8 +264,12 @@ if [[ -d $initdir/lib/modules/$kernel ]]; then
 fi
 
 if [[ $include_src && $include_target ]]; then
-    mkdir -p "$initdir$include_target"
-    cp -a -t "$initdir$include_target" "$include_src"/*
+    if [[ -f $include_src ]]; then
+        inst $include_src $include_target
+    else
+        mkdir -p "$initdir$include_target"
+        cp -a -t "$initdir$include_target" "$include_src"/*
+    fi
 fi
 
 for item in $install_items; do
