From 0a5ac37bc30495a9ce5175995e7c4b6718eab812 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 4 May 2011 10:17:46 +0200
Subject: [PATCH] iscsi: find iscsi kernel modules by symbol names

---
 modules.d/95iscsi/installkernel | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/modules.d/95iscsi/installkernel b/modules.d/95iscsi/installkernel
index ef069f8..2695f84 100755
--- a/modules.d/95iscsi/installkernel
+++ b/modules.d/95iscsi/installkernel
@@ -1,2 +1,8 @@
 #!/bin/bash
-instmods iscsi_tcp crc32c iscsi_ibft be2iscsi bnx2 bnx2x bnx2i
+instmods iscsi_tcp iscsi_ibft crc32c
+iscsi_module_test() {
+    local iscsifuncs='iscsi_register_transport'
+    fgrep -q "$iscsifuncs" "$1"
+}
+instmods $(filter_kernel_modules iscsi_module_test)
+
