From 112cd50ebae1037b8dfe78ed42806edf103e1476 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 5 Aug 2011 13:30:57 +0200
Subject: [PATCH] 90dm/dm-pre-udev.sh: load dm-mirror module

---
 modules.d/90dm/dm-pre-udev.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/90dm/dm-pre-udev.sh b/modules.d/90dm/dm-pre-udev.sh
index dfa4541..7da705d 100644
--- a/modules.d/90dm/dm-pre-udev.sh
+++ b/modules.d/90dm/dm-pre-udev.sh
@@ -1 +1,2 @@
 strstr "$(cat /proc/misc)" device-mapper || modprobe dm_mod 
+modprobe dm_mirror 2>/dev/null
