From 676a8c4262ef7733d9726011913ce5b118bbd819 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 6 Oct 2011 11:44:17 +0200
Subject: [PATCH] 10redhat-i18n: optionally install console_init

---
 modules.d/10redhat-i18n/install | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/10redhat-i18n/install b/modules.d/10redhat-i18n/install
index e7323a4..a74e648 100755
--- a/modules.d/10redhat-i18n/install
+++ b/modules.d/10redhat-i18n/install
@@ -94,7 +94,7 @@ else
     dracut_install gzip bzip2
 fi
 
-dracut_install /lib/udev/console_init 
+[ -x /lib/udev/console_init ] && dracut_install /lib/udev/console_init 
 dracut_install setfont loadkeys
 inst_rules 10-console.rules
 inst_hook cmdline 20 "$moddir/parse-i18n.sh"
