From 9db27c467915bec14697d4b58ad69db41217884f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 7 Oct 2011 09:05:41 +0200
Subject: [PATCH] TEST-40-NBD: relax check for fsoptions

---
 test/TEST-40-NBD/test.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/TEST-40-NBD/test.sh b/test/TEST-40-NBD/test.sh
index 55aacd5..1d79507 100755
--- a/test/TEST-40-NBD/test.sh
+++ b/test/TEST-40-NBD/test.sh
@@ -38,7 +38,7 @@ client_test() {
     local found opts nbdinfo
 
     [[ $fstype ]] || fstype=ext3
-    [[ $fsopt ]] || fsopt="errors=continue"
+    [[ $fsopt ]] || fsopt="ro"
 
     echo "CLIENT TEST START: $test_name"
 
