From 76904bf6d2230dc6469b80dbe7da01db4e69c61f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 10 Oct 2011 13:32:30 +0200
Subject: [PATCH] iscsi: install iscsi_boot_sysfs kernel module

---
 modules.d/95iscsi/installkernel | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/95iscsi/installkernel b/modules.d/95iscsi/installkernel
index 9806a5a..e90f662 100755
--- a/modules.d/95iscsi/installkernel
+++ b/modules.d/95iscsi/installkernel
@@ -1,5 +1,5 @@
 #!/bin/bash
-instmods iscsi_tcp iscsi_ibft crc32c bnx2i
+instmods iscsi_tcp iscsi_ibft crc32c bnx2i iscsi_boot_sysfs
 iscsi_module_test() {
     local iscsifuncs='iscsi_register_transport'
     fgrep -q "$iscsifuncs" "$1"
