From 655d9cb4a35095f9f2831e5efb9b315342694270 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 14 Feb 2012 14:07:30 +0100
Subject: [PATCH] 90mdraid/mdcontainer_start.sh: do not start with "-I"

---
 modules.d/90mdraid/mdcontainer_start.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90mdraid/mdcontainer_start.sh b/modules.d/90mdraid/mdcontainer_start.sh
index 46340a2..c92bfa7 100755
--- a/modules.d/90mdraid/mdcontainer_start.sh
+++ b/modules.d/90mdraid/mdcontainer_start.sh
@@ -4,7 +4,7 @@
 md=$1
 udevadm control --stop-exec-queue
 # and activate any containers
-mdadm -IR $md 2>&1 | vinfo
+mdadm -R $md 2>&1 | vinfo
 ln -s /sbin/mdraid-cleanup /pre-pivot/30-mdraid-cleanup.sh 2>/dev/null
 ln -s /sbin/mdraid-cleanup /pre-pivot/31-mdraid-cleanup.sh 2>/dev/null
 udevadm control --start-exec-queue
