From dd8d25ba3ee3157874ec0319aeedc2d4f6add970 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 1 Mar 2012 17:59:57 +0100
Subject: [PATCH] lsinitrd: require "file" and test for xz

---
 dracut.spec | 1 +
 lsinitrd    | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/dracut.spec b/dracut.spec
index 375fad1..9e53453 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -53,6 +53,7 @@ Requires: tar
 Requires: udev
 Requires: util-linux-ng >= 2.16
 Requires: which
+Requires: file
 
 %if ! 0%{?with_switch_root}
 Requires: util-linux-ng >= 2.16
diff --git a/lsinitrd b/lsinitrd
index 11e5c7a..0ee827a 100755
--- a/lsinitrd
+++ b/lsinitrd
@@ -36,7 +36,7 @@ image="${1:-/boot/initramfs-$(uname -r).img}"
 CAT=zcat
 FILE_T=$(file "$image")
 
-if echo "test"|xz|xz -dc --single-stream >/dev/null 2>&1; then
+if command -v xz >/dev/null && echo "test"|xz|xz -dc --single-stream >/dev/null 2>&1; then
     XZ_SINGLE_STREAM="--single-stream"
 fi
 
