From 2396effc41b86150d6efef7c6972e10d87141be7 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 2 Mar 2012 11:07:00 +0100
Subject: [PATCH] 99base/dracut-lib.sh: wait_for_if_up() increase wait time to
 10s

---
 modules.d/99base/dracut-lib.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/99base/dracut-lib.sh b/modules.d/99base/dracut-lib.sh
index 0887bd2..42d4dce 100644
--- a/modules.d/99base/dracut-lib.sh
+++ b/modules.d/99base/dracut-lib.sh
@@ -176,7 +176,7 @@ udevproperty() {
 
 wait_for_if_up() {
     local cnt=0
-    while [ $cnt -lt 20 ]; do 
+    while [ $cnt -lt 100 ]; do 
 	li=$(ip link show $1)
 	[ -z "${li##*state UP*}" ] && return 0
 	sleep 0.1
