From 15097b4ab984e660472f7a93779be532e20cbaa6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 28 Mar 2012 15:45:08 +0200
Subject: [PATCH] TEST-40-NBD: add check for nbd-server binary

---
 test/TEST-40-NBD/test.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/test/TEST-40-NBD/test.sh b/test/TEST-40-NBD/test.sh
index 1d79507..a6e7897 100755
--- a/test/TEST-40-NBD/test.sh
+++ b/test/TEST-40-NBD/test.sh
@@ -285,6 +285,10 @@ make_server_root() {
 }
 
 test_setup() {
+    if ! command -v nbd-server >/dev/null; then
+	echo "'nbd-server' not found"
+	return 1
+    fi
     make_encrypted_root || return 1
     make_client_root || return 1
     make_server_root || return 1
