From 1587cc77247a4b1984a2c31a17ae802daf6a752f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 28 Mar 2012 15:46:23 +0200
Subject: [PATCH] TEST-50-MULTINIC: refine error message

---
 test/TEST-50-MULTINIC/test.sh | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/test/TEST-50-MULTINIC/test.sh b/test/TEST-50-MULTINIC/test.sh
index 5bfdb75..5762863 100755
--- a/test/TEST-50-MULTINIC/test.sh
+++ b/test/TEST-50-MULTINIC/test.sh
@@ -49,15 +49,19 @@ client_test() {
   	-net nic,macaddr=52:54:00:12:34:$mac2,model=e1000 \
   	-net nic,macaddr=52:54:00:12:34:$mac3,model=e1000 \
         -net socket,connect=127.0.0.1:12345 \
-        -hdc /dev/null \
   	-kernel /boot/vmlinuz-$KVERSION \
   	-append "$cmdline $DEBUGFAIL rd_retry=5 rdinitdebug rdinfo rdnetdebug ro quiet console=ttyS0,115200n81 selinux=0 rdcopystate" \
   	-initrd initramfs.testing
 
-    if [[ $? -ne 0 ]] || ! grep -m 1 -q OK client.img; then
+    if [[ $? -ne 0 ]]; then
 	echo "CLIENT TEST END: $test_name [FAILED - BAD EXIT]"
 	return 1
     fi
+    if ! grep -m 1 -q OK client.img; then
+	echo "CLIENT TEST END: $test_name [FAILED - NO OK]"
+	return 1
+    fi
+
 
 
     for i in $check ; do
