From cfe414dce8af7c4221ea721dd56f18eb2505b57d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 29 Mar 2012 10:05:58 +0200
Subject: [PATCH] iscsi/iscsiroot: fixed iface.iscsi_ifacename param

---
 modules.d/95iscsi/iscsiroot | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/95iscsi/iscsiroot b/modules.d/95iscsi/iscsiroot
index dc93f7e..6faac58 100755
--- a/modules.d/95iscsi/iscsiroot
+++ b/modules.d/95iscsi/iscsiroot
@@ -155,7 +155,7 @@ handle_netroot()
 	${iscsi_password+-w $iscsi_password} \
 	${iscsi_in_username+-U $iscsi_in_username} \
 	${iscsi_in_password+-W $iscsi_in_password} \
-	${iscsi_iface_name+--param iface.iscsiname=$iscsi_iface_name} \
+	${iscsi_iface_name+--param iface.iscsi_ifacename=$iscsi_iface_name} \
 	${iscsi_netdev_name+--param iface.net_ifacename=$iscsi_netdev_name} \
 	${iscsi_param} \
 	|| :
