From e92ca555bff7333420ac043275cc5d6719c07233 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 19 Oct 2012 10:37:42 +0200
Subject: [PATCH] dracut.spec: add 02-fips.conf with 'do_strip=no'

---
 dracut.spec | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dracut.spec b/dracut.spec
index 7b2eaa7..8cf470c 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -189,6 +189,8 @@ rm $RPM_BUILD_ROOT/sbin/lsinitrd
 mkdir -p $RPM_BUILD_ROOT/etc/logrotate.d
 install -m 0644 dracut.logrotate $RPM_BUILD_ROOT/etc/logrotate.d/dracut
 
+echo 'do_strip=no' > $RPM_BUILD_ROOT/etc/dracut.conf.d/02-fips.conf
+
 %clean
 rm -rf $RPM_BUILD_ROOT
 
@@ -255,6 +257,7 @@ rm -rf $RPM_BUILD_ROOT
 %defattr(-,root,root,0755)
 %doc COPYING
 %{_datadir}/dracut/modules.d/01fips
+%config(noreplace) /etc/dracut.conf.d/02-fips.conf
 
 %files fips-aesni
 %defattr(-,root,root,0755)
