From 8485c3205f4c340b50f51849d3b4e7b0f3576883 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 19 Oct 2012 10:38:07 +0200
Subject: [PATCH] dracut: only 'warn' not 'error', if we don't strip

---
 dracut | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dracut b/dracut
index 3c3a926..0d63fcd 100755
--- a/dracut
+++ b/dracut
@@ -301,7 +301,7 @@ ldconfig -r "$initdir" || [[ $(id -u) != "0" ]] && dinfo "ldconfig might need ui
 if [[ $do_strip = yes ]] ; then
     for p in strip grep find; do 
 	if ! which $p >/dev/null 2>&1; then
-	    derror "Could not find '$p'. You should run $0 with '--nostrip'."
+	    dwarn "Could not find '$p'. Not stripping binaries in the initramfs."
 	    do_strip=no
 	fi
     done
