From a77208565ddb0a9d995b1303e899ad05f823f9da Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 27 Sep 2012 10:18:29 +0200
Subject: [PATCH] dracut.sh: create the initramfs non-world readable

---
 dracut | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/dracut b/dracut
index 0d63fcd..12be718 100755
--- a/dracut
+++ b/dracut
@@ -330,11 +330,11 @@ if strstr "$modules_loaded" " fips " && command -v prelink >/dev/null; then
 fi
 
 type pigz &>/dev/null && gzip=pigz || gzip=gzip
-( cd "$initdir"; find . |cpio -R 0:0 -H newc -o --quiet |$gzip -9 > "$outfile"; ) 
-if [ $? -ne 0 ]; then
+if ! ( umask 077; cd "$initdir"; find . |cpio -R 0:0 -H newc -o --quiet| \
+    $gzip -9 > "$outfile"; ); then
     derror "dracut: creation of $outfile failed"
     exit 1
-fi 
+fi
 dinfo "Wrote $outfile"
 dinfo $(ls -l "$outfile")
 
