From 5f99314801784cd18db152504faf5e1a9dc516bb Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 12 Feb 2013 10:22:27 +0100
Subject: [PATCH] base/initqueue: exit 0

---
 modules.d/99base/initqueue | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/99base/initqueue b/modules.d/99base/initqueue
index cc5ae37..ae9f8ba 100755
--- a/modules.d/99base/initqueue
+++ b/modules.d/99base/initqueue
@@ -39,4 +39,5 @@ fi
 
 mv -f "/tmp/$$-${job}.sh" "/initqueue${queue}/${job}.sh"
 [ -z "$queue" ] && >> /initqueue/work
+exit 0
 
