From 81134e54ad94886366cee9666e6c92e64462d6e9 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 29 Aug 2013 14:31:27 +0200
Subject: [PATCH] fcoe: honor "autovlan=yes"

---
 modules.d/95fcoe/fcoe-up | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/95fcoe/fcoe-up b/modules.d/95fcoe/fcoe-up
index b782f11..43a366e 100755
--- a/modules.d/95fcoe/fcoe-up
+++ b/modules.d/95fcoe/fcoe-up
@@ -36,7 +36,7 @@ if [ "$dcb" = "dcb" ]; then
     dcbtool sc "$netif" app:fcoe e:1 a:1 w:1
     sleep 1
     fipvlan "$netif" -c -s -f "-fcoe"
-elif [ "$netdriver" = "bnx2x" ]; then
+elif [ "$netdriver" = "bnx2x" ] || getarg 'autovlan=yes'; then
     # If driver is bnx2x, do not use /sys/module/fcoe/parameters/create but fipvlan
     modprobe 8021q
     udevadm settle --timeout=30 >/dev/null 2>&1
