From b9ec6563e7b559f2b3bb3c45818c2432a5af8fe6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 2 Oct 2013 12:29:41 +0200
Subject: [PATCH] add /etc/redhat-fips

---
 dracut.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/dracut.spec b/dracut.spec
index 10a1f90..c1bdf37 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -190,6 +190,7 @@ mkdir -p $RPM_BUILD_ROOT/etc/logrotate.d
 install -m 0644 dracut.logrotate $RPM_BUILD_ROOT/etc/logrotate.d/dracut
 
 echo 'do_strip=no' > $RPM_BUILD_ROOT/etc/dracut.conf.d/02-fips.conf
+> $RPM_BUILD_ROOT/etc/redhat-fips
 
 %clean
 rm -rf $RPM_BUILD_ROOT
@@ -260,6 +261,7 @@ rm -rf $RPM_BUILD_ROOT
 %doc COPYING
 %{_datadir}/dracut/modules.d/01fips
 %config(noreplace) /etc/dracut.conf.d/02-fips.conf
+%config(missingok) /etc/redhat-fips
 
 %files fips-aesni
 %defattr(-,root,root,0755)
