From de176906ff667f93b35f7f6d2e3c58b1dddec0b7 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 2 Oct 2013 12:29:53 +0200
Subject: [PATCH] rename /etc/redhat-fips to /etc/system-fips

---
 dracut.spec | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dracut.spec b/dracut.spec
index c1bdf37..47ed965 100644
--- a/dracut.spec
+++ b/dracut.spec
@@ -190,7 +190,7 @@ mkdir -p $RPM_BUILD_ROOT/etc/logrotate.d
 install -m 0644 dracut.logrotate $RPM_BUILD_ROOT/etc/logrotate.d/dracut
 
 echo 'do_strip=no' > $RPM_BUILD_ROOT/etc/dracut.conf.d/02-fips.conf
-> $RPM_BUILD_ROOT/etc/redhat-fips
+> $RPM_BUILD_ROOT/etc/system-fips
 
 %clean
 rm -rf $RPM_BUILD_ROOT
@@ -261,7 +261,7 @@ rm -rf $RPM_BUILD_ROOT
 %doc COPYING
 %{_datadir}/dracut/modules.d/01fips
 %config(noreplace) /etc/dracut.conf.d/02-fips.conf
-%config(missingok) /etc/redhat-fips
+%config(missingok) /etc/system-fips
 
 %files fips-aesni
 %defattr(-,root,root,0755)
