From afd17a3ec577f7677ef19d826b838d74d00d8c15 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 4 Nov 2013 17:17:31 +0100
Subject: [PATCH] fips: install /etc/system-fips and additional hmac files

---
 modules.d/01fips/install | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/modules.d/01fips/install b/modules.d/01fips/install
index 67ef284..95c41e1 100755
--- a/modules.d/01fips/install
+++ b/modules.d/01fips/install
@@ -20,13 +20,15 @@ else
         /$libdir/libfreebl3.so /$libdir/libfreebl3.chk 
 fi
 
-for i in /{usr,}/$libdir/libssl.so* /{usr,}/$libdir/libcrypto.so*; do
+for i in /{usr,}/$libdir/libssl.so* /{usr,}/$libdir/libcrypto.so* \
+         /{usr,}/$libdir/.libssl.so*.hmac /{usr,}/$libdir/.libcrypto.so*.hmac; do
     [ -e $i ] || continue
     dracut_install $i
 done
 
-dracut_install /usr/$libdir/hmaccalc/sha512hmac.hmac
+dracut_install -o /usr/$libdir/hmaccalc/sha*hmac.hmac
 dracut_install fipscheck
+dracut_install /etc/system-fips
 
 if command -v prelink >/dev/null; then
     dracut_install prelink
