From dd9b5df3b9c161a7b49c3d705989f68b8dfa9d85 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 8 Nov 2013 15:06:18 +0100
Subject: [PATCH] fips: fix RHEV vmlinuz check

---
 modules.d/01fips/fips.sh | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/modules.d/01fips/fips.sh b/modules.d/01fips/fips.sh
index 8d8fc73..7578bc6 100755
--- a/modules.d/01fips/fips.sh
+++ b/modules.d/01fips/fips.sh
@@ -57,7 +57,7 @@ do_rhevh_check()
     kpath=${1}
 
     # If we're on RHEV-H, the kernel is in /dev/.initramfs/live/vmlinuz0
-    HMAC_SUM_ORIG=$(cat /boot/.vmlinuz-${KERNEL}.hmac | while read a b; do printf "%s\n" $a; done)
+    HMAC_SUM_ORIG=$(cat $NEWROOT/boot/.vmlinuz-${KERNEL}.hmac | while read a b; do printf "%s\n" $a; done)
     HMAC_SUM_CALC=$(sha512hmac $kpath | while read a b; do printf "%s\n" $a; done || return 1)
     if [ -z "$HMAC_SUM_ORIG" ] || [ -z "$HMAC_SUM_CALC" ] || [ "${HMAC_SUM_ORIG}" != "${HMAC_SUM_CALC}" ]; then
         warn "HMAC sum mismatch"
@@ -81,10 +81,10 @@ do_fips()
         return 1
     fi
 
-    if [ -e "$NEWROOT/dev/.initramfs/live/vmlinuz0" ]; then
-        do_rhevh_check $NEWROOT/dev/.initramfs/live/vmlinuz0 || return 1
-    elif [ -e "$NEWROOT/dev/.initramfs/live/isolinux/vmlinuz0" ]; then
-        do_rhevh_check $NEWROOT/dev/.initramfs/live/isolinux/vmlinuz0 || return 1
+    if [ -e "/dev/.initramfs/live/vmlinuz0" ]; then
+        do_rhevh_check /dev/.initramfs/live/vmlinuz0 || return 1
+    elif [ -e "/dev/.initramfs/live/isolinux/vmlinuz0" ]; then
+        do_rhevh_check /dev/.initramfs/live/isolinux/vmlinuz0 || return 1
     else
         sha512hmac -c "/boot/.vmlinuz-${KERNEL}.hmac" || return 1
     fi
