From 1d85a6fe6d99429cc292056ef2521e5c0c4d5b58 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 27 Nov 2013 13:16:00 +0100
Subject: [PATCH] multipath: add rd_NO_MULTIPATH option

---
 dracut.8                            | 5 +++++
 modules.d/90multipath/multipathd.sh | 2 +-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/dracut.8 b/dracut.8
index 200c2ab..95bd0a3 100644
--- a/dracut.8
+++ b/dracut.8
@@ -260,6 +260,11 @@ disable DM RAID detection
 only activate the raid sets with the given UUID.
 This parameter can be specified multiple times.
 
+.SS MULTIPATH
+.TP
+.BR rd_NO_MULTIPATH
+disable multipath in the initramfs
+
 .SS FIPS
 .TP
 .BR fips=1
diff --git a/modules.d/90multipath/multipathd.sh b/modules.d/90multipath/multipathd.sh
index 2f8460f..e90d027 100755
--- a/modules.d/90multipath/multipathd.sh
+++ b/modules.d/90multipath/multipathd.sh
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-if [ -e /etc/multipath.conf ]; then
+if ! getarg rd_NO_MULTIPATH && [ -e /etc/multipath.conf ]; then
     modprobe dm-multipath
     multipathd -B || multipathd
 else
