From f22abb4b3e639989a3ab778760e44e200d529684 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 8 Jan 2015 12:53:44 +0100
Subject: [PATCH] dracut-lib: error out on empty parm for parse_iscsi_root()

---
 modules.d/99base/dracut-lib.sh | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/modules.d/99base/dracut-lib.sh b/modules.d/99base/dracut-lib.sh
index 171b860..89f66e7 100644
--- a/modules.d/99base/dracut-lib.sh
+++ b/modules.d/99base/dracut-lib.sh
@@ -285,9 +285,15 @@ killproc() {
 parse_iscsi_root()
 {
     local v
-    v=${1#iscsi:}
+    v=${1#iscsi}
+    v=${v#:}
 
-# extract authentication info
+    if [ -z "$v" ]; then
+        warn "parse_iscsi_root '$1' with invalid paramater"
+        return 1
+    fi
+
+    # extract authentication info
     case "$v" in
 	*@*:*:*:*:*)
 	    authinfo=${v%%@*}
