From d3fd23a1e1af98d5d78675cf2aba72948cbc497f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 3 Mar 2015 12:31:17 +0100
Subject: [PATCH] 45ifcfg/write-ifcfg.sh: clarify DHCPV6 case

---
 modules.d/45ifcfg/write-ifcfg.sh | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/modules.d/45ifcfg/write-ifcfg.sh b/modules.d/45ifcfg/write-ifcfg.sh
index 3d4b7be..5e570df 100644
--- a/modules.d/45ifcfg/write-ifcfg.sh
+++ b/modules.d/45ifcfg/write-ifcfg.sh
@@ -45,8 +45,7 @@ for netif in $IFACES ; do
         echo "ONBOOT=yes"
         echo "NETBOOT=yes"
         if [ -f /tmp/net.$netif.lease ]; then
-            strstr "$ip" '*:*:*' &&
-            echo "DHCPV6C=yes"
+            strstr "$ip" '*:*:*' && echo "DHCPV6C=yes"
             echo "BOOTPROTO=dhcp"
         else
             echo "BOOTPROTO=none"
