From 284c8be768d31b05249026f21ed6f8c8cf087f20 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 3 Mar 2015 18:19:44 +0100
Subject: [PATCH] netroot: do not bail out early

---
 modules.d/40network/netroot | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/modules.d/40network/netroot b/modules.d/40network/netroot
index cde039e..2bd4b82 100755
--- a/modules.d/40network/netroot
+++ b/modules.d/40network/netroot
@@ -15,8 +15,6 @@ PATH=$PATH:/sbin:/usr/sbin
 # or root is already there
 [ -e /tmp/root.info ] || exit 1
 . /tmp/root.info
-[ -d $NEWROOT/proc ] && exit 0
-[ -z "$netroot" ] && exit 1
 
 # Let's see if we have to wait for other interfaces
 # Note: exit works just fine, since the last interface to be 
@@ -89,6 +87,9 @@ if [ -z "$dest" ] ; then
 fi
 [ -n "$dest" ] && arping -q -f -w 60 -I $netif $dest
 
+[ -d $NEWROOT/proc ] && exit 0
+[ -z "$netroot" ] && exit 1
+
 # Check: do we really know how to handle (net)root?
 [ -z "$root" ] && die "No or empty root= argument"
 [ -z "$rootok" ] && die "Don't know how to handle 'root=$root'"
