From ec3fd83dc1d55ed6dc5f35262c2280f6c0f19b1b Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 3 Mar 2015 18:20:01 +0100
Subject: [PATCH] TEST-40-NBD: check for nbd kernel module

---
 test/TEST-40-NBD/test.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/test/TEST-40-NBD/test.sh b/test/TEST-40-NBD/test.sh
index 29d3e9d..4d163c2 100755
--- a/test/TEST-40-NBD/test.sh
+++ b/test/TEST-40-NBD/test.sh
@@ -289,6 +289,10 @@ test_setup() {
 	echo "'nbd-server' not found"
 	return 1
     fi
+    if ! modinfo nbd &>/dev/null; then
+        echo "Kernel module nbd missing"
+        return 1
+    fi
     make_encrypted_root || return 1
     make_client_root || return 1
     make_server_root || return 1
