From 1e7a5c2617b132113f47671a3255a8809908cbb6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 8 May 2015 13:47:29 +0200
Subject: [PATCH] iscsi: install timeout jobs for every iscsi netroot

---
 modules.d/95iscsi/parse-iscsiroot.sh | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/modules.d/95iscsi/parse-iscsiroot.sh b/modules.d/95iscsi/parse-iscsiroot.sh
index 1f6f982..66e6cc1 100755
--- a/modules.d/95iscsi/parse-iscsiroot.sh
+++ b/modules.d/95iscsi/parse-iscsiroot.sh
@@ -83,7 +83,10 @@ else
     # also call iscsiroot for misconfigured "ip=" or link down or MAC addr change
     # or replaced NIC
     # in the timeout queue
-    initqueue --onetime --timeout /sbin/iscsiroot dummy "$netroot" "$NEWROOT"
+    for nroot in $(getargs netroot); do
+        [ "${nroot%%:*}" = "iscsi" ] || continue
+        initqueue --onetime --timeout /sbin/iscsiroot dummy "$nroot" "$NEWROOT"
+    done
 fi
 
 netroot_enc=$(str_replace "$netroot" '/' '\2f')
