From 7b991e970262ca93c45a822941ffdeee167a76f0 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 8 May 2015 13:47:11 +0200
Subject: [PATCH] .dir-locals.el: fixup

---
 .dir-locals.el | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/.dir-locals.el b/.dir-locals.el
index c9ac3af..cbbcec4 100644
--- a/.dir-locals.el
+++ b/.dir-locals.el
@@ -1,2 +1,3 @@
-(setq sh-basic-offset: 4)
-(setq indent-tabs-mode nil)
+((nil . ((setq sh-basic-offset: 4)
+         (setq indent-tabs-mode nil)
+         )))
