From 3860f2788108339ef5de9667e0472d1323c13439 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 28 May 2015 13:56:44 +0200
Subject: [PATCH] ibft: correct device name

---
 modules.d/40network/parse-ibft.sh | 41 +++++++++++++++++++++++++++------------
 1 file changed, 29 insertions(+), 12 deletions(-)

diff --git a/modules.d/40network/parse-ibft.sh b/modules.d/40network/parse-ibft.sh
index 2c2cdae..80fe43b 100755
--- a/modules.d/40network/parse-ibft.sh
+++ b/modules.d/40network/parse-ibft.sh
@@ -19,6 +19,8 @@ for p in $(getargs ip=); do
             unset mask
             unset hostname
             unset vlan
+            unset dns1
+            unset dns2
             [ -e ${iface}/mac ] || continue
             ifname_mac=$(read a < ${iface}/mac; echo $a)
             [ -z "$ifname_mac" ] && continue
@@ -29,6 +31,7 @@ for p in $(getargs ip=); do
                     break
                 fi
             done
+
             if [ -z "$dev" ]; then
                 ifname_if=ibft$num
                 num=$(( $num + 1 ))
@@ -36,31 +39,45 @@ for p in $(getargs ip=); do
                 dev=$ifname_if
             fi
 
-            [ -e ${iface}/dhcp ] && dhcp=$(read a < ${iface}/dhcp; echo $a)
-            if [ -n "$dhcp" ]; then
-                echo "ip=$dev:dhcp"
-            else
-                [ -e ${iface}/ip-addr ] && ip=$(read a < ${iface}/ip-addr; echo $a)
-                [ "$ip" = "0.0.0.0" ] && unset ip
-                [ -e ${iface}/gateway ] && gw=$(read a < ${iface}/gateway; echo $a)
-                [ -e ${iface}/subnet-mask ] && mask=$(read a < ${iface}/subnet-mask; echo $a)
-                [ -e ${iface}/hostname ] && hostname=$(read a < ${iface}/hostname; echo $a)
-                [ -n "$ip" ] && echo "ip=$ip::$gw:$mask:$hostname:$dev:none"
-            fi
-
             if [ -e ${iface}/vlan ]; then
                 vlan=$(read a < ${iface}/vlan; echo $a)
                 if [ "$vlan" -ne "0" ]; then
                     case "$vlan" in
                         [0-9]*)
+                            [ -e /tmp/net.${dev}.${vlan}.has_ibft_config ] && continue
                             echo "vlan=$dev.$vlan:$dev"
+                            dev="${dev}.${vlan}"
                             ;;
                         *)
+                            [ -e /tmp/net.${vlan}.has_ibft_config ] && continue
                             echo "vlan=$vlan:$dev"
+                            dev="${vlan}"
                             ;;
                     esac
                 fi
             fi
+
+            [ -e /tmp/net.${dev}.has_ibft_config ] && continue
+
+            [ -e ${iface}/dhcp ] && dhcp=$(read a < ${iface}/dhcp; echo $a)
+
+            if [ -n "$dhcp" ]; then
+                echo "ip=$dev:dhcp"
+            else
+                [ -e ${iface}/ip-addr ] && ip=$(read a < ${iface}/ip-addr; echo $a)
+                [ "$ip" = "0.0.0.0" ] && unset ip
+                [ -e ${iface}/gateway ] && gw=$(read a < ${iface}/gateway; echo $a)
+                [ -e ${iface}/subnet-mask ] && mask=$(read a < ${iface}/subnet-mask; echo $a)
+                [ -e ${iface}/hostname ] && hostname=$(read a < ${iface}/hostname; echo $a)
+                [ -n "$ip" ] && echo "ip=$ip::$gw:$mask:$hostname:$dev:none"
+
+                [ -e ${iface}/primary-dns ] && dns1=$(read a < ${iface}/primary-dns; echo $a) \
+                                                   && echo "nameserver=${dns1}"
+                [ -e ${iface}/secondary-dns ] && dns2=$(read a < ${iface}/secondary-dns; echo $a) \
+                                                   && echo "nameserver=${dns2}"
+            fi
+
+            echo $mac > /tmp/net.${dev}.has_ibft_config
         done
     ) >> /etc/cmdline
     # reread cmdline
