From 5b276e801514b7729efb1bab486286e36cc0a455 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 13 Jan 2016 14:12:24 +0100
Subject: [PATCH] plymouth: add hyperv_fb kernel module

---
 modules.d/50plymouth/installkernel | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/50plymouth/installkernel b/modules.d/50plymouth/installkernel
index 9f3291d..632f45a 100755
--- a/modules.d/50plymouth/installkernel
+++ b/modules.d/50plymouth/installkernel
@@ -3,3 +3,5 @@
 for modname in $(find "$srcmods/kernel/drivers/gpu/drm" -name '*.ko' 2>/dev/null); do
     grep -q drm_crtc_init  $modname && instmods $modname
 done
+
+hostonly='' instmods hyperv_fb
