From e953f9e68893c75c32196b0a073c299196250b48 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 13 Jan 2016 14:20:13 +0100
Subject: [PATCH] network/ifup: create bond interface, if it does not yet exist

Resolves: rhbz#1263013
---
 modules.d/40network/ifup | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/40network/ifup b/modules.d/40network/ifup
index f91900e..942dfe8 100755
--- a/modules.d/40network/ifup
+++ b/modules.d/40network/ifup
@@ -151,6 +151,7 @@ if [ -e /tmp/bond.info ]; then
 
     if [ "$netif" = "$bondname" ] && [ ! -e /tmp/net.$bondname.up ] ; then # We are master bond device
         modprobe bonding
+        echo "+$netif" >  /sys/class/net/bonding_masters
         ip link set $netif down
 
         # Stolen from ifup-eth
