From 8d32f7965f6d772d07a8fb2e76b082d68e1f7545 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 19 Jan 2016 19:44:54 +0100
Subject: [PATCH] nfs/install: also add group "nobody" for rpc.idmapd

---
 modules.d/95nfs/install | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/95nfs/install b/modules.d/95nfs/install
index cc2a678..2a524fb 100755
--- a/modules.d/95nfs/install
+++ b/modules.d/95nfs/install
@@ -42,6 +42,7 @@ egrep '^rpc:' /etc/passwd >> "$initdir/etc/passwd"
 egrep '^rpcuser:' /etc/passwd >> "$initdir/etc/passwd"
 #which nologin >/dev/null 2>&1 && dracut_install nologin
 egrep '^rpc:' /etc/group >> "$initdir/etc/group"
+egrep '^nobody:' /etc/group >> "$initdir/etc/group"
 
 # rpc user needs to be able to write to this directory to save the warmstart
 # file
