From 44678ee9895f93d56ef8322bd1b14a0827f9bbfd Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 18 Feb 2016 15:45:07 +0100
Subject: [PATCH] dracut: add strglob

---
 modules.d/99base/dracut-lib.sh | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/modules.d/99base/dracut-lib.sh b/modules.d/99base/dracut-lib.sh
index 5ce329c..88fc9c6 100644
--- a/modules.d/99base/dracut-lib.sh
+++ b/modules.d/99base/dracut-lib.sh
@@ -22,6 +22,20 @@ str_replace() {
     echo "${out}${in}"
 }
 
+# returns OK if $1 matches (completely) glob pattern $2
+# An empty $1 will not be considered matched, even if $2 is * which technically
+# matches; as it would match anything, it's not an interesting case.
+strglob() {
+    [ -n "$1" -a -z "${1##$2}" ]
+}
+
+# returns OK if $1 contains (anywhere) a match of glob pattern $2
+# An empty $1 will not be considered matched, even if $2 is * which technically
+# matches; as it would match anything, it's not an interesting case.
+strglobin() {
+    [ -n "$1" -a -z "${1##*$2*}" ]
+}
+
 getarg() {
     set +x 
     local o line
