From e318a113bed042803de7c076a5436bd1b67b7744 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 10 Mar 2016 11:44:09 +0100
Subject: [PATCH] fix typo

---
 modules.d/95iscsi/iscsiroot | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/95iscsi/iscsiroot b/modules.d/95iscsi/iscsiroot
index 049bb6d..e444121 100755
--- a/modules.d/95iscsi/iscsiroot
+++ b/modules.d/95iscsi/iscsiroot
@@ -54,7 +54,7 @@ if getarg iscsi_firmware && ! [ -f /tmp/iscsistarted-firmware ]; then
     fi
 
     exit 0
-if
+fi
 
 handle_netroot()
 {
