From 6414f18c34b7a2b4ef07847670464773b9210d3a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 10 Mar 2016 14:34:04 +0100
Subject: [PATCH] network/ifup: fix vlan get_vid()

---
 modules.d/40network/ifup | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules.d/40network/ifup b/modules.d/40network/ifup
index 9f8fc6f..1cfed52 100755
--- a/modules.d/40network/ifup
+++ b/modules.d/40network/ifup
@@ -101,10 +101,10 @@ do_static() {
 get_vid() {
     case "$1" in
         vlan*)
-            return ${1#vlan}
+            echo ${1#vlan}
             ;;
         *.*)
-            return ${1##*.}
+            echo ${1##*.}
             ;;
     esac
 }
